src/css/application.css

```diff
@@ -566,7 +566,7 @@ body {
   position: absolute;
   top: 50%;
   left: 50%;
-  height: 20%;
+  height: 50%;
```

Comment:
Not that I could tell? I'd manually verified in dev tools that this property had no effect on display for Gist import. Are there other UI configurations that this is used for?